### PR TITLE
neuron model, data loading, and select/display input parameters

### PR DESCRIPTION
**Pull request by fkang-pu**
Completed the following tasks:

- Created a model "Neuron"

- Loaded 300 neurons

- datafile: example_data/worm_neuron_list.tsv
  copied from aconnectome_ids.txt(wormfunconn package)

- Added code to test.py for loading neurons.

- Downloaded css and js from "vanillaSelectBox" (https://github.com/PhilippeMarcMeyer/vanillaSelectBox), and use them for selecting neurons with search function.

- Created Django form named "ParamForm" to store/verify parameters required as input for wormfunconn. 

- Created template "list_input_params.html" for entering input parameters. Included JavaScript code to fill stim_neu_id and resp_neu_ids in the form with selected values. 

Notes:
- Passed SuperLinter locally

- Still need a lot of work on webpage layout. The above tasks were mainly aimed at creating a workflow for passing input values to the ParamForm.

- to-do: 
  add validation code to ParamForm and call wormfunconn functions to generate and show plot.

### CODE REVIEW

**Review by jcmatese — APPROVED** [review]
Seems ok to me.  Mused on using fixtures for the neuron data, but seems like roughly same amount of work, no matter what (and the other work has already been done).  Not certain why `neurondb.sqlite3` is a file to be committed to his repo, and I don't know if we need the full `vanillaSelectbox` example data and demos, but it is not a big deal.

**Review by lparsons — APPROVED** [review]
Overall this looks like a great start. There are a few things I'd clean up before merging and a few that we can fix later.

Before merging:

* [ ] Fix linting issues (many fixed in my PR)
* [ ] The neurondb.sqlite3 database file should be part of `.gitignore` and not checked into the repository
* [ ] Omit the modified vanillaSelectBox from linting

Fix later:
* [ ] Get installation of worm-functional-connectivity to work properly
* [ ] Get tests running in GitHub actions (likely fixed by above)

File: neuronsimulator/views.py [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+        # print(request.POST)
+        if my_form.is_valid():
+            stim_neuron = my_form.cleaned_data["stim_neuron"]
+            resp_neurons = my_form.cleaned_data["resp_neurons"]
+            nt = my_form.cleaned_data["nt"]
+            dt = my_form.cleaned_data["dt"]
+            stim_type = my_form.cleaned_data["stim_type"]
+            dur = my_form.cleaned_data["dur"]
+
+            # TODO: will add code for validations
```
Comment (hepcat72) [inline neuronsimulator/views.py:31]: C. Just a note that the validation code doesn't necessarily go here.  You can override an `is_valid` method in the form class (if I recall correctly).  Those methods are checked automatically.  And if this was a class, there would be a pair of methods here that say what to do if the form submission was (or wasn't) valid.  I'm not sure how that works in a function though.  There are also widgets for form validation that happens before form submission that are handled by form management.  You can insert labels for each field that show what errors to display for which fields where and how you want them to appear (though I haven't quite gotten the hang of the ins and outs of that yet).

**Review by hepcat72 — APPROVED** [review]
Not much to add beyond the other reviews.

**Comment by fkang-pu** [comment]
I pushed a commit to fix the errors related to linting. Now Superlinter passed without any error.  The error for functional test will be addressed later as suggested by Lance.  Most of the lines fixed in Lance's PR4 were also included in my new commit, except I have to keep the variable naming I used in Java script (oneNeuronChoice and multiNeuronsChoices in list_input_params.html without using dash) to avoid errors.  

My plan is to merge this PR (PR3) to main and close PR4 tomorrow morning, unless you have serious concern(s).  For suggestions you made in the past a few days, I will take into account when adding/modifying code to forms.py, views.py as well as related template file(s) in my new branch.

**Pull request merged**